common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -31,8 +31,14 @@
         </div>
     }
 
+
```

Comment:
NIT: are these spaces on purpose? Usually one or two spaces between things is enough